language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -19,6 +20,10 @@ use move_binary_format::{
 #[derive(Debug, Clone, Default)]
 pub struct VerifierConfig {
     pub max_loop_depth: Option<usize>,
+    pub treat_friend_as_private: bool,
```

Comment:
This config seam like do not use.

Reply:
Oh that's then a merge conflict. We should address this in a followup PR.